incuna-sass/helpers/_link.sass

```diff
@@ -1,18 +1,19 @@
 // Mixin to quickly add / remove default styles to links
 @mixin link($type: default)
     cursor: pointer
-    @if $type = default
-        text-decoration: underline
-        &:hover
-            text-decoration: none
     @if $type = reverse
         text-decoration: none
         &:hover
             text-decoration: underline
-    @if $type = blank
-        text-decoration: none
-        &:hover
+    @else
```

Comment:
it's not wrong, you have to check each argument one at a time, or it treats them as separate conditions. Then it ends up outputting multiple sets of code that just overwrite each other. 

@wytrych I don't really understand the difference between putting `@else @if` on the same line and what I did? How is this better than nesting?